cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -24,3 +28,126 @@ def test_cluster_builder(mocker):
     )
     print(yaml.dump(generated_template))
     # TODO assert content of the template by matching expected template
+
+
+@pytest.mark.parametrize(
+    "config_file_name",
+    [
+        "dashboard_config_1.yaml",
```

Comment:
let's use more self-explicative names. I'd add at least the scheduler in the file name.

Reply:
Done